src/stores/SearchStore.ts

```diff
@@ -486,6 +486,8 @@ export class SearchStore {
 		const bookId = this.booksStore.selectedBook.name;
 		const scope = this.searchForm.scope;
 
+		this.searchForm.stream = getArrayOfUniques(this.searchForm.stream);
```

Comment:
Not the best approach, I think its better to check for unic sessions when we just inserted session name into input, not when we initiate the search

Reply:
Also I'm not sure if we can with the same approach insert, for example, a bunch of filters with the same name, maybe here we also need to check for unique values